packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -3980,6 +3980,160 @@
   "guide_nova_explanations_arg3_text": "A multitude of sequences of processes is used to combine the usually many ingredients and to create the final product (hence 'ultra-processed'). The processes include several with no domestic equivalents, such as hydrogenation and hydrolysation, extrusion and moulding, and pre-processing for frying.",
   "guide_nova_explanations_arg4_title": "The predatory and financial aspects of ultra-processing",
   "guide_nova_explanations_arg4_text": "The overall purpose of ultra-processing is to create branded, convenient (durable, ready to consume), attractive (hyper-palatable) and highly profitable (low-cost ingredients) food products designed to displace all other food groups. Ultra-processed food products are usually packaged attractively and marketed intensively.",
+  "guide_nova_share_link": "https://en.openfoodfacts.org/nova",
```

Comment:
Could you add a comment, that says "translate only if the link exists" (or something like)